src/renderer/components/alert/RpcStatusAlert.tsx

```diff
@@ -0,0 +1,27 @@
+import { Alert } from '@material-ui/lab';
+import { CircularProgress } from '@material-ui/core';
+import { useAppSelector } from '../../../store/hooks';
+import { RpcProcessStateType } from '../../../models/rpcModel';
+
+export default function RpcStatusAlert() {
+  const rpcProcess = useAppSelector((s) => s.rpc.process);
+  if (rpcProcess.type === RpcProcessStateType.STARTED) {
+    return (
+      <Alert severity="warning" icon={<CircularProgress size={22} />}>
+        The swap daemon is starting
+      </Alert>
+    );
+  }
+  if (rpcProcess.type === RpcProcessStateType.LISTENING_FOR_CONNECTIONS) {
+    return <Alert severity="success">The swap daemon is running</Alert>;
+  }
+  if (rpcProcess.type === RpcProcessStateType.NOT_STARTED) {
+    return <Alert severity="warning">The swap daemon is being started</Alert>;
+  }
+  if (rpcProcess.type === RpcProcessStateType.EXITED) {
+    return (
+      <Alert severity="error">The swap daemon has stopped unexpectedly</Alert>
+    );
+  }
```

Comment:
**Suggestion:** Consider adding a default case to the conditionals that check the `rpcProcess.type`. This would handle any unexpected values and make the code more robust.
```suggestion
  if (rpcProcess.type === RpcProcessStateType.STARTED) {
    ...
  } else if (rpcProcess.type === RpcProcessStateType.LISTENING_FOR_CONNECTIONS) {
    ...
  } else if (rpcProcess.type === RpcProcessStateType.NOT_STARTED) {
    ...
  } else if (rpcProcess.type === RpcProcessStateType.EXITED) {
    ...
  } else {
    return <Alert severity="error">Unknown RPC process state</Alert>;
  }
```